docs/content/en/docs/core-components/reporting.md

```diff
@@ -14,13 +14,13 @@ SOARCA utilizes push-based reporting to provide information on the instantiation
 
```

Comment:
Maybe we should add a paragraph of flow diagram that explains the asynchronous nature of reporting.